public/images/pages/regtest-sm.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="65" height="65" viewBox="0 0 65 65" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
do I see correctly that the icon is the same as for the testnet? could we just reuse that?

Reply:
You're right, and we could. I just made the separation so we can more easily make updates to this SVG in the future to have different icons between the two networks.